scenarios/api.github.com/get-archive/normalized-fixture.json

```diff
@@ -49,17 +49,20 @@
     "responseIsBinary": true,
     "headers": {
       "access-control-allow-origin": "https://render.githubusercontent.com",
+      "age": "0",
       "connection": "close",
       "content-disposition": "attachment; filename=octokit-fixture-org-get-archive-0000000.tar.gz",
       "content-security-policy": "default-src 'none'; style-src 'unsafe-inline'; sandbox",
       "content-type": "application/x-gzip",
       "date": "Tue, 10 Oct 2017 16:00:00 GMT",
       "etag": "\"00000000000000000000000000000000\"",
       "strict-transport-security": "max-age=31536000",
+      "via": "1.1 varnish-v4",
       "x-content-type-options": "nosniff",
       "x-frame-options": "deny",
       "x-geo-block-list": "",
       "x-github-request-id": "0000:00000:0000000:0000000:00000000",
+      "x-varnish": "74624297",
```

Comment:
`x-varnish` header is set to a request id that is meant to help find requests in varnish logs, it must be normalized like other request IDs